adal/src/Microsoft.IdentityModel.Clients.ActiveDirectory/Internal/Http/HttpClientWrapper.cs

```diff
@@ -141,7 +142,7 @@ public async Task<IHttpWebResponse> GetResponseAsync()
                         string.Format(CultureInfo.CurrentCulture,
                             "Response status code does not indicate success: {0} ({1}).",
                             (int) webResponse.StatusCode, webResponse.StatusCode),
-                        new AdalException(webResponse.ResponseString)));
```

Comment:
Minor refactoring: consolidate HttpResponse (MSAL) and HttpWebResponse (ADAL) under a single interface IWebResponse (in core).